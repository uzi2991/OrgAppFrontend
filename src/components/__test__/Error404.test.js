import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from 'react-router-dom';
import "@testing-library/jest-dom";
import Error404 from "../../pages/Error404";

test("Error404 snapshot", () => {
    const errorSnapshot = renderer.create(
        <Router>
            <Error404 />
        </Router>

    ).toJSON();

    expect(errorSnapshot).toMatchSnapshot();
})

