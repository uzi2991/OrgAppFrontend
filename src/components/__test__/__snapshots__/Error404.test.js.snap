// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 snapshot 1`] = `
<div
  className="error-page"
>
  <img
    className="error-page__image"
    src="404.jpg"
  />
  <div
    className="error-page__content"
  >
    <h1
      className="error-page__title"
    >
      You look a little lost there
    </h1>
    <h4
      className="error-page__subtitle"
    >
      This page may be private. You may be able to view it by
       
      <a
        className="error-page__link"
        href="/login"
        onClick={[Function]}
      >
        logging in
      </a>
    </h4>
  </div>
</div>
`;
